pylearn2/scripts/tests/test_dbm.py

```diff
@@ -0,0 +1,105 @@
+"""
+Tests scripts in the DBM folder
+"""
+
+import cPickle
+import os
+import pylearn2.scripts.dbm.show_negative_chains as negative_chains
+import pylearn2.scripts.dbm.show_reconstructions as reconstructions
+import pylearn2.scripts.dbm.show_samples as samples
+import pylearn2.scripts.dbm.top_filters as top_filters
+from pylearn2.config import yaml_parse
+from pylearn2.models.dbm.layer import BinaryVector, BinaryVectorMaxPool
+from pylearn2.datasets.mnist import MNIST
+from pylearn2.models.dbm.dbm import DBM
+from nose.tools import with_setup
+from pylearn2.datasets import control
+from pylearn2.utils import serial
+
+
+def setup():
+    """Create pickle file with a simple model."""
+    control.push_load_data(False)
```

Comment:
Oh, is tearDown guaranteed to run? The pop_load_data in the tearDown works then.
It might be nice to put a comment here saying the push is paired with a pop in the tearDown.